templates/frontOffice/default/includes/add-to-cart-error.html

```diff
@@ -0,0 +1,21 @@
+{* This page should not replace the current previous URL *}
```

Comment:
I'm not sure this change need to be in this PR

Reply:
File has been removed from PR